CoinDock-app/src/App/Api/signup.js

```diff
@@ -0,0 +1,51 @@
+import baseApi from "./api";
+import { getUserId } from "App/Auth/helper";
+const signup = baseApi.injectEndpoints({
+  endpoints: (build) => ({
+    postRegister: build.mutation({
+      query: ({
+        firstname,
+        lastname,
+        date,
+        email,
+        country,
+        password,
+        reenterpassword,
+        ...data
+      }) => ({
+        url: "/v1/users",
+        method: "post",
+        data: {
+          first_name: firstname,
+          last_name: lastname,
+          date_of_birth: date,
+          email: email,
+          country: country,
+          password: password,
+          re_enter_password: reenterpassword,
+        },
+      }),
+      transformResponse: (response) => {
```

Comment:
Since from here there is no modification is done so i think there is no need of this transform response

Reply:
removed